doc/src/manual/environment-variables.md

```diff
@@ -65,6 +65,15 @@ and a global configuration search path of
 /etc/julia/startup.jl
 ```
 
+### `JULIA_PROJECT`
+
+A directory path that points to the current Julia project. Setting this
+environment variable has the same effect as specifying the `--project` start-up
+option, but `--project` has higher precedence.  If the variable is set to `@.`,
+Julia tries to find a project directory that contains Project.toml or
+JuliaProject.toml file from the current directory and its parents.  See also
```

Comment:
I'd suggest wraping the toml filenames in backticks, to match the convention used in this document for paths.

Reply:
Thanks. Fixed.